imports/ui/components/Card.jsx

```diff
@@ -141,7 +141,6 @@ const useStyles = theme => ({
     }
   },
   details: {
-    display: "flex",
```

Comment:
With ```display: "flex"``` it will look like this though: 
![image](https://user-images.githubusercontent.com/17561746/62185202-577d6980-b316-11e9-8b06-82fb3e3f9ee0.png)

But removed looks like this: 
![image](https://user-images.githubusercontent.com/17561746/62185216-6b28d000-b316-11e9-8322-8467db2633a2.png)

Hmmm. Fix one thing, break another...